pkg/controller/volume/persistentvolume/pv_controller_test.go

```diff
@@ -273,9 +284,7 @@ func TestControllerSync(t *testing.T) {
 		},
 	}
 
-	for _, test := range tests {
-		klog.V(4).Infof("starting test %q", test.name)
-
+	doit := func(test controllerTest) {
```

Comment:
Why not this?
```go
for _, test := range tests {
    test := test
    t.Run(test.name, func(t *testing.T) {
```

Reply:
Because I didn't want to re-indent the entire test code. As it stands now, "git blame" is still useful, whereas without the `doit` function everything would get shifted once to the right.